vector/src/main/java/im/vector/app/features/home/ShortcutsHandler.kt

```diff
@@ -87,6 +96,25 @@ class ShortcutsHandler @Inject constructor(
                 .launchIn(coroutineScope)
     }
 
+    fun updateShortcutsWithPreviousIntent() {
+        // Check if it's been already done
+        if (sharedPreferences.getBoolean(SHARED_PREF_KEY, false)) return
+        ShortcutManagerCompat.getShortcuts(context, ShortcutManagerCompat.FLAG_MATCH_PINNED)
+                .filter { it.intent.component?.className == RoomDetailActivity::class.qualifiedName }
+                .mapNotNull {
+                    it.intent.getStringExtra("EXTRA_ROOM_ID")?.let { roomId ->
```

Comment:
Previously defined in `RoomDetailActivity` with `const val EXTRA_ROOM_ID = "EXTRA_ROOM_ID"` but I prefer to use hard-coded string here, since we are removing it.